cos_registration_server/api/serializer.py

```diff
@@ -62,3 +67,35 @@ def validate_grafana_dashboards(self, value):
                 "gafana_dashboards is not a supported format (list)."
             )
         return dashboards
+
+    def validate_foxglove_layouts(self, value):
+        """Validate foxglove layouts data.
+
+        value: Foxglove layouts provided data.
+        return: Foxglove layouts as dict.
+        raise:
+          json.JSONDecodeError
+          serializers.ValidationError
+        """
+        if isinstance(value, str):
```

Comment:
This kind of things makes me think we should start using mypy in this project.

Reply:
In this case, mypy would be only partly beneficial. The serializer gets data from the request. These data can be strings containing a dict or a dict directly.

Reply:
It would still help understanding what gets in & out of those functions, especially in PRs like this one until I have a better in-depth view of the overall project.

Reply:
Mypy is already enabled but types are not added: https://github.com/ubuntu-robotics/cos-registration-server/blob/main/Makefile#L44

Reply:
Yeah I would suggest to do that :+1:

Reply:
I will add the types in another PR. Otherwise this one is going to explode.